virtual_rainforest/models/litter/constants.py

```diff
@@ -73,3 +134,48 @@ class LitterConsts:
     The value given here is taken from :cite:t:`fatichi_mechanistic_2019`; see
     documentation for :attr:`cue_metabolic` for details.
     """
+
+    cue_woody: float = 0.55
+    """Carbon use efficiency of woody litter decay [unitless].
+
+    The value given here is taken from :cite:t:`fatichi_mechanistic_2019`; see
+    documentation for :attr:`cue_metabolic` for details.
+    """
+
+    cue_structural_below_ground: float = 0.45
+    """Carbon use efficiency of belowground structural litter decay [unitless].
+
+    The value given here is taken from :cite:t:`fatichi_mechanistic_2019`; see
+    documentation for :attr:`cue_metabolic` for details.
+    """
+
+    depth_of_active_layer: float = 0.25
+    """Depth of the biogeochemically active soil layer [m].
+
+    The soil model considered a homogenous layer in which all significant nutrient
+    processes take place. This is a major assumption of the model. The value is taken
+    from :cite:t:`fatichi_mechanistic_2019`. No empirical source is provided for this
+    value.
+
+    This is really a core constant as it is shared across models. However, the core
+    constants are not yet setup, so this constant is being stored here for the time
+    being.
+    """
+
+    air_entry_water_potential: float = -3.815
+    """Water potential at which soil pores begin to aerate [kPa].
+
+    The constant is used to estimate soil water potential from soil moisture. As this
+    estimation is a stopgap this constant probably shouldn't become a core constant. The
+    value is the average across all soil types found in
+    :cite:t:`cosby_statistical_1984`. In future, this could be calculated based on soil
+    texture.
+    """
+
+    water_retention_curvature: float = -7.22
```

Comment:
I agree that this is an important issue, we should definitely make sure that we coordinate these calculations across models. It might be that some of the hydrology moves to soil or that this functionality moves to the hydrology model. Or the plants. Either way, we don't want to duplicate constants or use slightly different approaches to estimate variables